### PR TITLE
[IMP]Payments terms TASK#4162, HU#959

### PR DESCRIPTION
**Pull request by hugho-ad**
Added payments terms based on qty of payments and based on date of payments

[TASK#4162](https://www.vauxoo.com/web#id=4162&view_type=form&model=project.task&action=138)

### CODE REVIEW

**Comment by hugho-ad** [comment]
@JulioSerna can you help me with review/+1 please

Regards

File: payment_term_type/models/account_settings.py [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+
+
+class AccountConfigSettings(models.TransientModel):
+    _inherit = 'account.config.settings'
+
+    payment_type = fields.Selection([
+        ('bqp', 'Based on quantity of payments'),
+        ('bdp', 'Based on date of payments')],
+        string="Payments terms",
+        help='*bqp.- Based on quantity of payments\n'
```
Comment (JulioSerna) [inline payment_term_type/models/account_settings.py:36]: cual seria el help para bdp?

Comment (hugho-ad) [inline payment_term_type/models/account_settings.py:36]: @JulioSerna había omitido esa parte del texto, ya esta corregido el mensaje del help

**Comment by JulioSerna** [comment]
:+1:

**Comment by hugho-ad** [comment]
@moylop260  can you help  me with a review/merge please.

Regards

**Pull request merged**